README.md

```diff
@@ -166,9 +166,24 @@ spec:
 | Private Builder Image Registry     | ⚪️     |      |    |
 | Runtime Base Image    | ⚪️    |      |    |
 | Binary builds         |       |      |    |
+| Image pull policy     |       |      |    |
+| Inject source via config map         |       |      |    |
+| Inject source via secret         |       |      |    |
+| Inject source via image         |       |      |    |
+| Set image label/ens   |       |      |    |
+| Triggers (SCMs)       |       |      |    |
+| Triggers (imagestream)|       |      |    |
+| Pruning of builds     |       |      |    |
+| Cancelling of builds  |       |      |    |
+| Chaining of builds    |       |      |    |
 | Image Caching         |       |      |    |
+| Max duration / TO     |       |      |    |
+| Parallel vs. serial execution   |       |      |    |
 | ImageStreams support  |       |      |    |
 | Entitlements          |       |      |    |
```

Comment:
@adambkaplan IMO same story as image cache ... the current CSI plugin choice for mutating annotated pods should work for buildv2 like buildv1 .... then it is a question of any OCM or openshift/builder image changes for buildv1 being pulled into the buildv2 controller